src/client/pages/Section/DataTable/Table/hooks/useParsedTable.ts

```diff
@@ -41,11 +43,19 @@ export const useParsedTable = (props: Props): Returned => {
   const { countryIso } = useCountryRouteParams<CountryIso>()
   const cycle = useCycle()
   const showODP = useShowOriginalDatapoints()
+  const country = useCountry(countryIso)
   const odpYears = useOriginalDataPointYearsWithHistory({ assessmentName, table: _table })
+  const historyLastApprovedIsActive = useHistoryLastApprovedIsActive()
+
   const { print } = useIsPrintRoute()
 
   return useMemo<Returned>(() => {
-    const _props = { assessmentName, countryIso, cycle, odpYears, showODP, table: _table, print }
+    const ignoreOdpYears =
```

Comment:
@yaguzmang when history is active in 1b and country doesn't use odp, odp columns should not be shown.
With this change they are. I'm about to push the fix on this branch